frontend/src/utils/utils.ts

```diff
@@ -15,6 +15,20 @@ export const getEtherscanUrl = (txHash: string, chainId: number) => {
   return `https://${networkPrefix}/tx/${txHash}`;
 };
 
+/**
+ * @notice Generates short names based on the given `chainId`
+ */
+export const getChainShortName = (chainId?: number) => {
```

Comment:
Can this method just return the `chainName` from the `supportedChains` list? Similar to the Etherscan comment above—that way when adding a network we can just do it all in the one file, and these methods will pull from there

Also maybe rename to just getChainName since we'll pull directly from that list

Reply:
On [Tornado](https://app.tornado.cash/) they have short name in the selected field and long names in dropdown.
If we keep the names short, there wouldn't be a need for this method at all.

Reply:
Good data point. I was looking at https://gnosis-safe.io/app/#/ which only uses short names. 

Personally I think the short names are sufficiently descriptive and I'd be ok removing this

Reply:
Maybe we wait to see what @apbendi thinks